lambda/js/src/main/scala/feral/lambda/facade/Context.scala

```diff
@@ -17,14 +17,17 @@
 package feral.lambda.facade
 
 import scala.scalajs.js
+import scala.scalajs.js._
 
 @js.native
 private[lambda] sealed trait Context extends js.Object {
   def callbackWaitsForEmptyEventLoop: Boolean = js.native
   def functionName: String = js.native
   def functionVersion: String = js.native
   def invokedFunctionArn: String = js.native
-  def memoryLimitInMB: String = js.native
+  // Note: This should always be a string, but the Netlify CLI currently passes a number.
+  // ref: https://github.com/netlify/functions/pull/251
```

Comment:
Upstreamed again to https://github.com/ashiina/lambda-local/pull/218 but I'm not optimistic.

Reply:
Fixed and released!